alpine/entrypoint/pre_start.sh

```diff
@@ -45,6 +45,12 @@ if [ -n "$LDAP_HOST" ]; then
   fi
 fi
 
+# Copy any existing configuration files before starting the container
+if [ -d "$ENTRYPOINT_HOME/pre_config" ]; then
```

Comment:
we could actually make this simpler by just mounting configs to `/config` i've been thinking about exposing most of the places that files/directories should be mounted as directories under `/` to make it simpler